src/Schema/EloquentSchema.php

```diff
@@ -155,6 +155,23 @@ protected function getDefaultAttributes(Model $model)
     }
 
     /**
+     * Get attributes for the provided model using fillable attribute.
+     *
+     * @param Model $model
+     * @return array
+     */
+    protected function getDefaultModelAttributes(Model $model)
+    {
+        $schemaAttributes = $this->attributes;
+        if(empty($schemaAttributes))
+        {
+            $schemaAttributes = $model->getFillable();
```

Comment:
If `$model->getVisible()` is not empty, that list should be used (as it is a white-list of serializable attributes).

If using `$model->getFillable()`, you'll need to remove any keys that are in `$model->getHidden()`.